client/src/includes/a11y-result.ts

```diff
@@ -8,8 +8,8 @@ import {
 
 const sortAxeNodes = (nodeResultA?: NodeResult, nodeResultB?: NodeResult) => {
   if (!nodeResultA || !nodeResultB) return 0;
-  const nodeA = document.querySelector<HTMLElement>(nodeResultA.target[0]);
-  const nodeB = document.querySelector<HTMLElement>(nodeResultB.target[0]);
+  const nodeA = document.querySelector<HTMLElement>(`${nodeResultA.target[0]}`);
+  const nodeB = document.querySelector<HTMLElement>(`${nodeResultB.target[0]}`);
```

Comment:
Note: These changes are needed due to axe-core's TS changes to `CrossTreeSelector` & `ShadowDomSelector`

> Argument of type 'CrossTreeSelector' is not assignable to parameter of type 'string'.
> Type 'ShadowDomSelector' is not assignable to type 'string'.ts(2345)